preprocessing_qc/make_pseudobulk.ipynb

```diff
@@ -0,0 +1,959 @@
+{
```

Comment:
<p>added clarification: "Since we are combining single cell and atac, we need to make sure that the bulk and pseudobulk values are on the same scale; meaning one dataset isn't 10 times larger than the other."</p>